editoast/src/modelsv2/fixtures.rs

```diff
@@ -310,3 +308,15 @@ pub async fn create_work_schedule_group(conn: &mut DbConnection) -> WorkSchedule
         .await
         .expect("Failed to create empty work schedule group")
 }
+
+pub fn get_trainschedule_json_array() -> &'static str {
+    include_str!("../tests/train_schedules/simple_array.json")
+}
+
+pub fn get_fast_rolling_stock_schema(name: &str) -> RollingStock {
+    let mut rolling_stock_form: RollingStock =
+        serde_json::from_str(include_str!("../tests/example_rolling_stock_1.json"))
+            .expect("Unable to parse");
+    rolling_stock_form.name = name.to_string();
+    rolling_stock_form
+}
```

Comment:
Why do we need these new functions in a PR that only change the `DbConnection`?

Reply:
Removed in https://github.com/OpenRailAssociation/osrd/pull/8712/commits/e8c0350489cfc4c6543056d5fb81ba53621c7e21.